python/solutions/part1/longest_word_finder.py

```diff
@@ -0,0 +1,9 @@
+sentence_string = "This is a long sentence with words of multiple lengths for inputting into your function"
+longest_word = ''
+
+words = sentence_string.split()
```

Comment:
for clarity, swap lines 2 and 4? I think of the splitting as something to be done right away with the sentence to get an array, and then the longest_word initialization should happen immediately before the loop to emphasize that it'll be changed in the loop.